scripts/deployBridge.ts

```diff
@@ -25,73 +26,66 @@ export async function deployL2Bridge(
   l2GovRelay: bigint,
```

Comment:
nitpick: l2GovRelay to be added as param to doc

Reply:
good catch!